.github/concierge.yaml

```
@@ -0,0 +1,25 @@
+juju:
+  channel: 3.6/stable
```

suggestion(non-blocking): increase logging level:

```suggestion
  channel: 3.6/stable
  model-defaults:
    logging-config: <root>=INFO; unit=DEBUG
```